workmail-message-flow-state-machine/README.md

```diff
@@ -0,0 +1,83 @@
+# Amazon WorkMail Message Flow State Machine
+
+This application enables you to use AWS Step Functions to orchestrate multiple AWS Lambda functions for Amazon WorkMail and implement additional business logic into your WorkMail message flow.
+
+## Setup
+1. Deploy this application via [AWS Serverless Application Repository](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:489970191081:applications~workmail-message-flow-state-machine).
+2. Open the [WorkMail Console](https://console.aws.amazon.com/workmail/) and create a synchronous **RunLambda** [Email Flow Rule](https://docs.aws.amazon.com/workmail/latest/adminguide/lambda.html#synchronous-rules) that uses this Lambda function.
```

Comment:
It will be worth explaining a little more about how the synchronous rule timeout plays out with step function execution time since the two are coupled together.

`ruleTimeout` should be the more than the max time step function will take, I believe.